boards/Kconfig.v2

```
@@ -1,9 +1,10 @@
-# Copyright (c) 2022 Nordic Semiconductor ASA
+# Copyright (c) 2022-2024 Nordic Semiconductor ASA
 
 # SPDX-License-Identifier: Apache-2.0
 
 BOARD_STRING := $(sanitize_upper,$(BOARD))
 BOARD_FULL_STRING := $(sanitize_upper,$(BOARD)$(BOARD_IDENTIFIER))
+BOARD_QUALIFIERS_WITHOUT_SEPARATOR := $(substring,$(BOARD_IDENTIFIER),1)
```

afaiu the board qualifiers is without the first separator.

```suggestion
BOARD_QUALIFIERS := $(substring,$(BOARD_IDENTIFIER),1)
```

We anyway need to update the CMake code according to #69905 3.a, and then we should simply just pass in the qualifiers as it has new been defined by the terminology discussion.

---

Fixed